pulp_container/app/registry_api.py

```diff
@@ -664,7 +664,7 @@ def put(self, request, path, pk=None):
             # Upload has been deleted => task has started or even finished
             task = Task.objects.filter(
                 name__endswith="add_and_remove",
-                reserved_resources_record=[f"upload:{pk}"],
+                reserved_resources_record__contains=[f"shared:upload:{pk}"],
```

Comment:
Help my memory please: We sneak the upload in the resources records because there is no other place we can annotate the task, right? And making it shared, it is the least intrusive?

Reply:
correct

Reply:
shipit!